units/Cargo.toml

```diff
@@ -17,6 +17,9 @@ default = ["std"]
 std = ["alloc", "internals/std"]
 alloc = ["internals/alloc"]
 
+# Only used for testing.
+test-serde = ["serde", "internals/test-serde"]
```

Comment:
No, this should've been `internals = { package = "bitcoin-internals", version = "0.3.0", features = ["test-serde"] }` inside `dev-dependencies` to make it not public.

Reply:
Can do, I had that locally at first.